docs/internals_gpio_control.md

```diff
@@ -6,17 +6,16 @@ QMK has a GPIO control abstraction layer which is microcontroller agnostic. This
 
 The following functions can provide basic control of GPIOs and are found in `quantum/quantum.h`.
 
-|Function              |Description                                                       |
-|----------------------|------------------------------------------------------------------|
-|`setPinInput(pin)`    |Set pin as input with high impedance (High-Z)                     |
-|`setPinInputHigh(pin)`|Set pin as input with build in pull-up                            |
-|`setPinInputLow(pin)` |Set pin as input with build in pull-down (Supported only on STM32)|
-|`setPinOutput(pin)`   |Set pin as output                                                 |
-|`writePinHigh(pin)`   |Set pin level as high, assuming it is an output                   |
-|`writePinLow(pin)`    |Set pin level as low, assuming it is an output                    |
-|`writePin(pin, level)`|Set pin level, assuming it is an output                           |
-|`readPin(pin)`        |Returns the level of the pin                                      |
-
+|Function              |Description                                                       | Old AVR Examples                               | Old ChibiOS/ARM Examples                      |
+|----------------------|------------------------------------------------------------------|------------------------------------------------|-----------------------------------------------|
+|`setPinInput(pin)`    |Set pin as input with high impedance (High-Z)                     | DDRB &= ~(1<<2)                                | palSetLineMode(pin, PAL_MODE_INPUT)           |
```

Comment:
Needs code tags, the pipe character messes up the table formatting :)

Reply:
lol, whoops, will fix.